### PR TITLE
Doc: Update isStandardTx comment

### PR DESCRIPTION
**Pull request by smatthewenglish**
Incremental improvement to readability.

### CODE REVIEW

**Comment by pstratem** [comment]
utACK 8d3a32fcd3fe4747c64bd09e344c0a22751784d8

**Comment by laanwj** [comment]
This could use a more descriptive commit message.

File: src/policy/policy.cpp [hunk 1]
```diff
@@ -78,7 +78,7 @@ bool IsStandardTx(const CTransaction& tx, std::string& reason)
         // bytes of scriptSig, which we round off to 1650 bytes for some minor
         // future-proofing. That's also enough to spend a 20-of-20
         // CHECKMULTISIG scriptPubKey, though such a scriptPubKey is not
-        // considered standard)
```
Comment (paveljanik) [inline src/policy/policy.cpp]: When touching this comment, can you please also move the '(520 byte limit)' above before the dot?

**Comment by paveljanik** [comment]
@s-matthew-english Can you please fix these nits, so we an proceed with this PR?

**Comment by s-matthew-english** [comment]
sure- but not until later today

**Comment by MarcoFalke** [comment]
Please squash and make the commit message something like `[doc] Improve IsStandardTx comment`

**Comment by s-matthew-english** [comment]
I'm new to contributing to projects on here, but I'm trying to learn, that's why I did this, so I'm wondering, what does it mean to squash the commit? How to do that? 

*Please forgive me if this question is daft.

**Comment by paveljanik** [comment]
See e.g. #1952 where git rebase -i is described. Or you are doing it via web (patch-2?).

**Comment by paveljanik** [comment]
But squashing these two commits into one can be done when merging, but better do this yourself.

**Comment by s-matthew-english** [comment]
cool- I'll do this tonight

**Comment by s-matthew-english** [comment]
so I tried to do it from my terminal- since originally I made these changes just online using GitHub web app- but- did the changes percolate? I used the suggested name of the commit message and all :/ 

Please forgive me if this is a bit daft- but- Im just learning how to become a contributor to an open source project.

**Comment by paveljanik** [comment]
Looks like you need to `git push` yet :-)

**Comment by s-matthew-english** [comment]
so then just 'git push -u origin master'?

**Comment by paveljanik** [comment]
I think `git push -f origin patch-2`

**Comment by s-matthew-english** [comment]
he tell me this:

"error: src refspec patch-2 does not match any."

**Comment by paveljanik** [comment]
Hmm, your current work is in `patch-2`, so it looks like to do not have a proper clone of it locally.
What does `git remote show origin` show for you?
Can you join IRC?

**Comment by s-matthew-english** [comment]
btc

**Comment by paveljanik** [comment]
Ah, so you cloned the original source, not your patch-2 branch. OK.
This should work then:

```
git remote add s-matthew-english https://www.github.com/s-matthew-english/bitcoin
git push -f s-matthew-english patch-2
```

**Comment by s-matthew-english** [comment]
ah, rats! seems- 
btddc
no dice!

**Comment by s-matthew-english** [comment]
http://stackoverflow.com/questions/5802426/git-error-src-refspec-master-does-not-match-any

^ I tried that :/
anyway- I'll keep werking on it

**Comment by paveljanik** [comment]
So please start from scratch in empty directory:
```
https://www.github.com/s-matthew-english/bitcoin
cd bitcoin
git checkout --track origin/patch-2
git rebase -i e9723cb27384ccd8e4bd78d2fea907534fd0d94a
... change pick to squash on the second line
... change the commit message
git push -f origin patch-2
```
OK?

**Comment by s-matthew-english** [comment]
Hmm- I don't know if it worked. 

I'm not so good at VIM

I tried to change the commit message after the step `git rebase -i e9723cb27384ccd8e4bd78d2fea907534fd0d94a`

but I don't know if these ones percolated:
... change pick to squash on the second line
... change the commit message

and then finally it just told me:

"Everything up-to-date"

So- that doesn't seem right :/

I'm sorry to be so daft believe me!

**Comment by s-matthew-english** [comment]
How to join IRC?

**Comment by paveljanik** [comment]
Join IRC here: https://bitcoincore.org/en/contribute/ - there is a webchat link to join irc.freenode.net.

**Comment by s-matthew-english** [comment]
for some reason the link http://irc.freenode.net/ seems not to load on
Chromium or Firefox

**Comment by s-matthew-english** [comment]
maybe my Uni blocks it?

**Comment by paveljanik** [comment]
Click on #bitcoin-core-dev.

**Comment by s-matthew-english** [comment]
ok cool- I'm there as 'matthew_english'

**Comment by s-matthew-english** [comment]
btc

**Comment by paveljanik** [comment]
ACK https://github.com/bitcoin/bitcoin/pull/7786/commits/03c77fdc143fb8d533011f23164daac560e381b2

**Comment by laanwj** [comment]
@s-matthew-english Thanks for going through this

@paveljanik Thanks for helping him go through this

ACK https://github.com/bitcoin/bitcoin/pull/7786/commits/03c77fdc143fb8d533011f23164daac560e381b2

**Pull request merged**